pkg/entrypoint/entrypointer.go

```diff
@@ -63,6 +64,8 @@ type Entrypointer struct {
 
 	// Results is the set of files that might contain task results
 	Results []string
+	//Timeout is an optional user-specified flag for timing out Steps
```

Comment:
nit: space after `//`. Suggest slightly different wording: `Timeout is an optional user-specified duration within which the Step must complete`